src/navigators/PopupWindow.ts

```diff
@@ -57,12 +57,15 @@ export class PopupWindow extends AbstractChildWindow {
     public async navigate(params: NavigateParams): Promise<NavigateResponse> {
         this._window?.focus();
 
-        const popupClosedInterval = setInterval(() => {
+        const popupClosedInterval = { id: setInterval(() => {
```

Comment:
Within this library we have multiple `setInterval` calls, and never as `{ id: setInterval...}` needed/used, please avoid this here as well

Reply:
Fixed.